src/classes/storage.ts

```diff
@@ -114,16 +114,34 @@ export class Storage {
 
     if (!groundPlace || groundPlace.type !== placeType) {
       throw new Error(
-        `The ${placeType} with the Gpuid ${placeToRemoveGpuid} cannot be deleted because it cannot be found.`,
+        `The "${placeType}" with the Gpuid "${placeToRemoveGpuid}" cannot be deleted because it cannot be found.`,
       );
     }
 
     if (groundPlace.childs.length) {
       throw new Error(
-        `The ${placeType} with the Gpuid ${placeToRemoveGpuid} cannot be deleted because it is not empty.`,
+        `The "${placeType}" with the Gpuid "${placeToRemoveGpuid}" cannot be deleted because it has children.`,
       );
     }
 
+    // Delete the place from the GroundPlaces list.
     this.groundPlaces.splice(placeIndex, 1);
+
+    // If the place is a StopGroup, all references to this StopGroup are also removed from its possible StopCluster parent.
+    if (placeType === GroundPlaceType.GROUP) {
```

Comment:
you'll need to create a removeStopGroupFromStopCluster function later.
Because it's also something we will be doing.

(verifying the stopGroup exists OR has at least another parent)

Reply:
Yep, this is planned for the next PR